packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -25,16 +39,32 @@ export default function SidebarNavigationScreenGlobalStyles() {
 			) }
 			content={ <StyleVariationsContainer /> }
 			actions={
-				<SidebarButton
-					icon={ edit }
-					label={ __( 'Edit styles' ) }
-					onClick={ () => {
-						// switch to edit mode.
-						setCanvasMode( 'edit' );
-						// open global styles sidebar.
-						openGeneralSidebar( 'edit-site/global-styles' );
-					} }
-				/>
+				<div>
```

Comment:
I've just used a simple div here so that there isn't any space between the buttons. Happy to change this if we need the buttons to have some breathing room

Reply:
Sounds good 👍 

I'm not an expert here, but would something like `<div role="group" aria-label="Styles commands">` be appropriate too?

Asking for a friend :trollface:

Reply:
Good question — I only added the wrapping `div` for styling purposes rather than to flag anything semantically. Since there's only two controls at the moment, I wasn't sure how valuable having a group role would be for those two buttons?

Happy to add it in, though, if you think it's better to treat them as a group!

Reply:
Honestly I don't know. I think it's fine to leave as is. If we find out a group role is helpful it's easy to add in 👍

Reply:
Cheers, sounds good 👍